consul-mixin/dashboards.libsonnet

```diff
@@ -82,14 +82,14 @@ local panel_settings = {
         g.row('Consul Server')
         .addPanel(
           g.panel('QPS') +
-          g.queryPanel('sum(rate(consul_http_request_count{job="$job"}[$__rate_interval])) by (instance)', '{{instance}}') +
+          g.queryPanel('sum(rate(consul_http_request_count{job=~"$job"}[$__rate_interval])) by (instance)', '{{instance}}') +
           g.stack
         )
         .addPanel(
           g.panel('Latency') +
-          g.queryPanel('max(consul_http_request{job="$job", quantile="0.99"})', '99th Percentile') +
-          g.queryPanel('max(consul_http_request{job="$job", quantile="0.5"})', '50th Percentile') +
-          g.queryPanel('sum(rate(consul_http_request{job="$job"}[5m])) / sum(rate(consul_http_request{job="$job"}[5m]))', 'Average') +
+          g.queryPanel('max(consul_http_request{job=~"$job", quantile="0.99"}) by (instance)', '99th Percentile', '{{instance}}') +
+          g.queryPanel('max(consul_http_request{job=~"$job", quantile="0.5"}) by (instance)', '50th Percentile', '{{instance}}') +
+          g.queryPanel('sum(rate(consul_http_request{job=~"$job"}[$__rate_interval])) by (instance) / sum(rate(consul_http_request{job=~"$job"}[$__rate_interval])) by (instance)', 'Average') +
```

Comment:
@tomwilkie This query was originally written by you. I've changed it to aggregate by instance (like the QPS query), and to use `$__rate_interval`. However, it looks like this is EXACTLY the same value, divided by itself, which will always be 1.

I'm not sure if this was meant to be two different metrics, like the query duration, divided by the query count, or if maybe an offset range was intended?

In any case, I think this query still needs to change. Can you provide insight?

Where do these metrics come from? They aren't in the consul exporter 🤔